src/ts/components/seekbar.ts

```diff
@@ -253,6 +259,10 @@ export class SeekBar extends Component<SeekBarConfig> {
       if (this.isUiShown) {
         this.setAriaSliderValues();
       }
+
+      if (typeof uimanager.getConfig().seekbarSnappingRange !== 'undefined') {
```

Comment:
```suggestion
      if (typeof uimanager.getConfig().seekbarSnappingRange === 'number') {
```
would make more sense IMO